frontend/src/theme/components/Field/Checkbox.ts

```diff
@@ -0,0 +1,78 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+import { getColor } from '@chakra-ui/theme-tools'
+
+/**
+ * This must be kept in line with the key from Chakra's internal
+ * Checkbox styling to ensure that the styles are merged correctly.
+ * https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/index.ts
+ */
+export const CHECKBOX_THEME_KEY = 'Checkbox'
+
+/**
+ * With reference to
+ * https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/checkbox.ts
+ */
+
+const parts = ['container', 'control', 'label', 'icon']
+
+export const Checkbox: ComponentMultiStyleConfig = {
+  parts,
+  baseStyle: ({ colorScheme: c, theme }) => ({
+    // Control is the box containing the check icon
+    control: {
+      borderRadius: '0.25rem',
+      border: '0.125rem solid',
+      borderColor: `${c}.500`,
+      // When the label is long and overflows to the next line, we want
+      // the checkbox to be aligned with the first line rather than the center
+      alignSelf: 'start',
+      _focus: {
+        boxShadow: 'none',
+      },
+      _disabled: {
+        borderColor: getColor(theme, `neutral.500`),
+        bg: 'white',
+        _checked: {
+          borderColor: getColor(theme, `neutral.500`),
+          bg: getColor(theme, `neutral.500`),
+        },
+      },
+    },
+    // Container for the checkbox as well as label
+    container: {
+      w: '100%',
+      px: '0.25rem',
+      py: '0.5rem',
+      _hover: {
```

Comment:
should we disable bg on hover on disabled state?

Reply:
if you say so hahaha